js/sales.js

```diff
@@ -8,80 +8,80 @@
         }
     }
 
     CookieStore.prototype.render = function(){
-        var tableStoreRow = document.createElement('tr');
+        const tableStoreRow = document.createElement('tr');
         tableStoreRow.appendChild(createDataCell(this.name, 'th', 'row'));
 
         this.calculateSimulatedCookiesPerHour();
         
-        for(var i = 0; i < this.simulatedCookiesPerHour.length; i++){
+        for(let i = 0; i < this.simulatedCookiesPerHour.length; i++){
             tableStoreRow.appendChild(createDataCell(this.simulatedCookiesPerHour[i], 'td'));
         }
         document.querySelector('tbody').appendChild(tableStoreRow);
         return tableStoreRow;
     }
 
     CookieStore.prototype.calculateStoreTotal = function(){
-        var total = 0;
-        for(var i = 0; i < this.simulatedCookiesPerHour.length; i++){
+        let total = 0;
+        for(let i = 0; i < this.simulatedCookiesPerHour.length; i++){
             total += this.simulatedCookiesPerHour[i];
         }
         return total;
     }
 
     function createStoreTableContents(){
-        var hoursTableRow = createHoursHeadRow();
+        const hoursTableRow = createHoursHeadRow();
         hoursTableRow.appendChild(createDataCell('Daily Location Total', 'th', 'col'));
 
-        var allStoresTotal = 0;
-        for(var i = 0; i < cookieStoreLocations.length; i++){
-            var store = cookieStoreLocations[i];
-            var storeTableRow = store.render();
-            var total = store.calculateStoreTotal();
+        let allStoresTotal = 0;
+        for(let i = 0; i < cookieStoreLocations.length; i++){
+            const store = cookieStoreLocations[i];
+            const storeTableRow = store.render();
+            const total = store.calculateStoreTotal();
             allStoresTotal += total;
             storeTableRow.appendChild(createDataCell(total, 'td'));
         }
-        var totalsRow = createTotalsFooterRow();
+        const totalsRow = createTotalsFooterRow();
         totalsRow.appendChild(createDataCell(allStoresTotal, 'td'));
     }
 
     function createHoursHeadRow(){
         //use military time
-        var openingHour = 6;
-        var standardTimeOffSet = 12;
+        const openingHour = 6;
+        const standardTimeOffSet = 12;
         
-        var tableRow = document.createElement('tr'); 
+        const tableRow = document.createElement('tr'); 
         tableRow.appendChild(createDataCell('Store Locations', 'th', 'col')).setAttribute('id', 'store-locations');
 
-        for(var i = 0; i < OPEN_HOURS; i++){
-            var am = ':00am';
-            var pm = ':00pm';
-            var textContent;
-            var militaryHour = openingHour + i;
+        for(let i = 0; i < OPEN_HOURS; i++){
+            const am = ':00am';
+            const pm = ':00pm';
+            let textContent;
+            let militaryHour = openingHour + i;
```

Comment:
Since this variable is recreated on every iteration of the loop (because `let` is scoped to the code block, unlike `var` which would be scoped to the whole function), we can safely use `const` for `militaryHour`. We never reassign the value of `militaryHour` within this code block.